build-logic/packaging/src/main/kotlin/bisq/gradle/packaging/JLinkTask.kt

```diff
@@ -27,46 +31,103 @@ abstract class JLinkTask : DefaultTask() {
+            logger.lifecycle("Executing jlink command: ${processBuilder.command().joinToString(" ")}")
+
+            // Add module path if specified
+            if (javaModulesDirectory.isPresent) {
+                val commands = processBuilder.command()
+                commands.add("--module-path")
+                commands.add(javaModulesDirectory.asFile.get().absolutePath)
+                logger.lifecycle("Executing jlink command: ${commands.joinToString(" ")}")
+            }
+
+            logger.lifecycle("Preparing process builder..")
+            processBuilder.inheritIO()
+
+            logger.lifecycle("Starting process builder..")
+            val process = processBuilder.start()
+            val reader =
+                BufferedReader(InputStreamReader(process.inputStream))
+            while ((reader.readLine()) != null) {
+                // WORKAROUND: Jlink hangs in windows (11pro) when running from this custom plugin
+                // This buffer reader is needed jus to get it going and won't affect the rest of the platforms
+            }
+            logger.lifecycle("Reading errors..")
+            val errorOutput = process.errorStream.bufferedReader().readText()
+            val exitCode = process.waitFor()
+
+            if (exitCode != 0) {
+                logger.error("jlink error output:\n$errorOutput")
+                throw IllegalStateException("jlink couldn't create custom runtime. Exit code: $exitCode")
+            }
         }
     }
 
     private fun parseUsedJavaModulesFromJDepsOutput(): String {
-        var readLines = jDepsOutputFile.asFile.get().readLines()
-        if (!javaModulesDirectory.isPresent) {
-            readLines = readLines.filter { it.startsWith("java.") }
-        }
-        return readLines.joinToString(",")
+//        TODO instead of hardcoding he modules process the jdeps report and generate the below so we don't need
+//             to touch this code again in case of needed new jmods in the future
+//            readLines.filter { pattern.matcher(it).matches() }
+//            .flatMap { line -> line.split("->").map { it.trim() } }
+//            .filter { it.startsWith("java.") || it == "bisq.desktop_app_launcher" }
+//            .flatMap { line -> line.split(",").map { it.trim() } }
+//            .distinct()
+//            .joinToString(",")
+//        val readLines = jDepsOutputFile.asFile.get().readLines()
+        val modules = listOf(
+            // base
+            "java.base",
+            "java.datatransfer",
+            "java.desktop",
+            "java.logging",
+            "java.xml",
+            "java.naming",
+            "java.net.http",
+            // security
+            "java.se",
+            "java.security.jgss",
+            // javafx
+            "javafx.base",
+            "javafx.controls",
+            "javafx.fxml",
+            "javafx.graphics",
+            "javafx.media",
+            "javafx.swing",
+            "javafx.web",
```

Comment:
I am not aware that we have any dependency to javafx.web

Reply:
great, I'll do a quick test to see if we can safely remove that - this will reduce the size of the final build even more :)

Reply:
thanks so much @HenrikJannsen , indeed the binary works perfectly without javafx.web mod and we save an impressive 100MB more being the total reduction ~46.14% now :dagger: :smile:

Reply:
Could you try how much difference removal of `javafx.swing` cause? We cannot remove it now as its used but not in major use cases. If reduction is large enough it would justify investigating how to replace the current usage (SystemTray, AwtImage,....).

Reply:
@HenrikJannsen not much of a difference, just a few kbs

Reply:
Ok, thanks. Then we leave it